cli/README.md

```diff
@@ -580,6 +580,7 @@ code-push release-cordova <appName> <platform>
 [--targetBinaryVersion <targetBinaryVersion>]
 [--rollout <rolloutPercentage>]
 [--build]
+[--isReleaseBuildType]
```

Comment:
Both the `--build` and `--isReleaseBuildType` flags need to be alphabetically ordered above